### PR TITLE
Admonition and text fixes

### PR DESCRIPTION
**Pull request by mmattel**
This PR fixes:
- fixed missed broken links
- using simple admonition prefix when possible
- line break fixes for better source text readbility
- fixing oc_app_command xref's

### CODE REVIEW

File: modules/administration_manual/pages/configuration/server/background_jobs_configuration.adoc [hunk 1]
```diff
@@ -73,7 +72,8 @@ executing:
 *  *  *  *  * /usr/bin/php -f /path/to/your/owncloud/cron.php
 ....
 
-NOTE: You have to make sure that `php` is found by `cron`, hence why we’ve deliberately added the full path to the PHP binary above (`/usr/bin/php`). On some systems it might be necessary to use *php-cli* instead of *php*.
+NOTE: You have to make sure that `php` is found by `cron`, hence why we’ve deliberately added the full path
```
Comment (settermjd) [inline modules/administration_manual/pages/configuration/server/background_jobs_configuration.adoc]: There is no need to format "php" and "cron" as inline source. Better to write them as PHP and Cron.

File: modules/administration_manual/pages/configuration/server/occ_app_command.adoc [hunk 2]
```diff
@@ -14,19 +14,19 @@ Please ensure that the app you are going to configure is installed and enabled.
...
-* xref:ocapps_contacts[Contacts]
-* xref:ocapps_files_antivirus[Anti-Virus]
-* xref:ocapps_files_primary_s3[S3 Objectstore]
-* xref:ocapps_ldap[LDAP]
-* xref:ocapps_market_commands[Market]
-* xref:ocapps_password_policy[Password Policy]
-* xref:ocapps_ransomware[Ransomware Protection]
-* xref:ocapps_samle-sso-sibboleth-integration[SAML/SSO Shibboleth Integration]
-* xref:ocapps_two_factor_auth[Two Factor Authentication]
+* xref:run-occ-as-your-http-user[Run occ As Your HTTP User]
```
Comment (settermjd) [inline modules/administration_manual/pages/configuration/server/occ_app_command.adoc]: 👍

File: modules/administration_manual/pages/configuration/user/user_auth_ldap.adoc [hunk 3]
```diff
@@ -667,13 +661,11 @@ server
 * The certificate is announced in the system’s LDAP configuration file,
 usually `/etc/ldap/ldap.conf`.
 
-* Using LDAPS, also make sure that the port is correctly configured (by
-default 636)
+* Using LDAPS, also make sure that the port is correctly configured (by default 636)
```
Comment (settermjd) [inline modules/administration_manual/pages/configuration/user/user_auth_ldap.adoc]: Please change the text in parenthesis to "the default port is 636".

File: modules/administration_manual/pages/configuration/user/user_auth_ldap.adoc [hunk 4]
```diff
@@ -667,13 +661,11 @@ server
...
 * If you get the error "_Lost connection to LDAP server_" or
 "**No connection to LDAP server**" double check the connection parameters and
 try connecting to LDAP with tools like `ldapsearch`. If using ldaps or
```
Comment (settermjd) [inline modules/administration_manual/pages/configuration/user/user_auth_ldap.adoc]: Please capitalise LDAPS, as you have done with LDAP.

File: modules/administration_manual/pages/enterprise/classification_and_policy_enforcement.adoc [hunk 5]
```diff
@@ -235,11 +235,10 @@ You can construct a workflow rule using the following steps:
 * Select the tag `Class: Confidential` to be added when the rules match.
 * Hit "_Add workflow_" to save and enable it.
 
-[NOTE]
-====
-For more information, please check the options available for auto-tagging and consult the xref:enterprise/file_management/files_tagging.adoc[Workflow extension Documentation].
-For files classified with the _Workflow_ extension, administrators can impose feature and access policies as described in the next section.
-====
+NOTE: For more information, please check the options available for auto-tagging and consult the
+xref:enterprise/file_management/files_tagging.adoc[Workflow extension Documentation].
```
Comment (settermjd) [inline modules/administration_manual/pages/enterprise/classification_and_policy_enforcement.adoc]: s/Workflow extension Documentation/Workflow Extension documentation/g

File: modules/administration_manual/pages/enterprise/firewall/file_firewall.adoc [hunk 6]
```diff
@@ -79,26 +77,22 @@ The request (ispublic share link|other) request.
...
-The request’s `REMOTE_ADDR` header (is|is not) matching the given IP
-range.
+The request’s `REMOTE_ADDR` header (is|is not) matching the given IP range.
 
 [[file-size-upload]]
 ==== File Size Upload
 
-When a file is uploaded the size has to be (lessgreater|greater or
-equal) to the given size.
+When a file is uploaded the size has to be (lessgreater|greater or equal) to the given size.
```
Comment (settermjd) [inline modules/administration_manual/pages/enterprise/firewall/file_firewall.adoc]: Is "lessgreater" a typo?

File: modules/administration_manual/pages/enterprise/firewall/file_firewall.adoc [hunk 7]
```diff
@@ -79,26 +77,22 @@ The request (ispublic share link|other) request.
...
 
 [[file-mimetype-upload]]
 ==== File Mimetype Upload
 
-When a file is uploaded the mimetype (isbegins withends with|does not
-end with) the given string.
+When a file is uploaded the mimetype (isbegins withends with|does not end with) the given string.
```
Comment (settermjd) [inline modules/administration_manual/pages/enterprise/firewall/file_firewall.adoc]: Is "isbegins withends" a typo"?

File: modules/administration_manual/pages/configuration/server/background_jobs_configuration.adoc (hunk 1 above)
Comment (mmattel) [reply modules/administration_manual/pages/configuration/server/background_jobs_configuration.adoc]: ? please help wat should be written here

File: modules/administration_manual/pages/configuration/user/user_auth_ldap.adoc (hunk 4 above)
Comment (mmattel) [reply modules/administration_manual/pages/configuration/user/user_auth_ldap.adoc]: just to let you know, I did not change the content but linebreaking in the source for better readablity...

File: modules/administration_manual/pages/configuration/server/background_jobs_configuration.adoc (hunk 1 above)
Comment (settermjd) [reply modules/administration_manual/pages/configuration/server/background_jobs_configuration.adoc]: "NOTE: You have to make sure that PHP is found by Cron; hence why we’ve deliberately added the full path"

**Pull request merged**